OpenMarket/OpenMarket/Controller/ProductListViewController.swift

```diff
@@ -5,9 +5,22 @@
 
 import UIKit
 
-fileprivate enum SegmentIndex: Int {
+fileprivate enum SegmentIndex: Int, CaseIterable {
   case list = 0
   case grid
+  
+  var title: String {
+    switch self {
+    case .list:
+      return "LIST"
+    case .grid:
+      return "GRID"
+    }
+  }
+  
+  static var indexs: [String] {
```

Comment:
[typo] 
```suggestion
  static var indexes: [String] {
```
```suggestion
  static var indices: [String] {
```

사실상 인덱스가 아니라 title들의 배열인데요. 사용처에서 혼동이 있을 수 있을 것 같아요 
titles 와 같은 직관적인 네이밍은 어떨까요?

Reply:
혼동이 발생했던 네이밍이네요..!!
추천해주신 titles로 수정하였습니다.